src/main/java/ru/practicum/shareit/item/service/ItemService.java

```diff
@@ -69,4 +72,14 @@ public interface ItemService {
      * @param userId id пользователя, наличие которого необходимо проверить
      */
     void checkUserExistence(Long userId);
+
+    /**
+     * Добавляет комментарий
+     *
+     * @param userId id комментатора
+     * @param itemId id  предмета
+     * @param comment id удаляемого предмета
+     * @return Сообщение об удалении предмета
+     */
+    ru.practicum.shareit.comment.dto.CommentOutputDto addComment(Long userId, Long itemId, Comment comment);
```

Comment:
Нужно убрать полное название пути:
```suggestion
    CommentOutputDto addComment(Long userId, Long itemId, Comment comment);
```

Reply:
Исправил ошибку